lms/static/scripts/frontend_apps/components/BookPicker.tsx

```diff
@@ -234,8 +234,9 @@ export default function BookPicker({
       )}
       // Opt out of Modal's automatic focus handling; route focus manually in
       // sub-components
-      initialFocus={'manual'}
+      initialFocus="manual"
       onClose={onCancel}
+      closeTitle="Close book dialog"
```

Comment:
```suggestion
      closeTitle="Close book picker"
```

I suggest we align the title with what we call the widget in code and in the KB to avoid confusion.